pkg/kwokctl/runtime/kind/cluster.go

```diff
@@ -52,6 +53,20 @@ func NewCluster(name, workdir string) (runtime.Runtime, error) {
 	}, nil
 }
 
+// Available  checks whether the runtime is available.
+func (c *Cluster) Available(ctx context.Context) error {
+	err := exec.Exec(context.Background(), "", exec.IOStreams{}, consts.RuntimeTypeKind)
+	if err != nil {
+		return err
+	}
+
+	// kind depends on docker
+	// TODO: nerdctl kind provider support is discussing in
+	// https://github.com/kubernetes-sigs/kind/issues/2317 and
+	// https://github.com/containerd/nerdctl/issues/349
+	return exec.Exec(context.Background(), "", exec.IOStreams{}, "docker")
```

Comment:
```suggestion
	return exec.Exec(ctx, "", exec.IOStreams{}, "docker")
```

Reply:
Currently Kind actually supports podman, so it's not really a good idea to check docker.

Reply:
I am not that familiar with kind, maybe we can check docker and podman, it is ok when one of them works?

or just leave it returning nil directly?

Reply:
Yeah, just check Kind.

Reply:
but kind would be downloaded by kwokctl if not existed...

Reply:
now I check docker or podman, because you mentioned kwokctl will download kind if not found, WDYT?

Reply:
Yeah, You're right, do as you say.